Add App to Teams/Add-TeamsAppInstallation.ps1

```diff
@@ -0,0 +1,24 @@
+﻿$logfile = "C:\TeamsAppInstallationlog_$(get-date -format `"yyyyMMdd_hhmmsstt`").txt"
+$start = [system.datetime]::Now
+
+Connect-MicrosoftTeams
+$AppName=Read-Host "Please enter AppName"
```

Comment:
consider turning these into required parameters, PowerShell will prompt for them if not provided